package.json

```diff
@@ -14,6 +14,7 @@
   },
   "packageManager": "pnpm@9.12.1",
   "engines": {
+    "//comment": "Update @types/node when updating this node version",
```

Comment:
Voor zover ik weet kan `.nvmrc` geen comments bevatten, vandaar het `"//comment" hier. Zou ook `"//"` kunnen zijn eventueel.
Andere oplossingen welkom!

Reply:
`$comment` is denk ik vrij gebruikelijk?